app.py

```
@@ -0,0 +1,323 @@
+        
+        process = subprocess.Popen(
+            cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True,
+            bufsize=1,
+            universal_newlines=True,
+            env={**os.environ, 'PYTHONUNBUFFERED': '1'}
+        )
+        
+        total_pages = 0
```

Ruff (0.8.2)

Local variable `total_pages` is assigned to but never used

Remove assignment to unused variable `total_pages`

(F841)